resources/lib/oojs-ui/oojs-ui.js

```diff
@@ -3187,6 +3188,9 @@ OO.ui.WindowManager.prototype.updateWindowSize = function ( win ) {
 		sizes = this.constructor.static.sizes,
 		size = win.getSize();
 
+	if ( size === 'dynamic' && typeof win.getDynamicSize === 'function' ) {
```

Comment:
do we have any tests for this part of code already in VE. seams like a good candidate for adding test